ballista/core/src/execution_plans/distributed_query.rs

```diff
@@ -278,7 +285,12 @@ async fn execute_query(
     query: ExecuteQueryParams,
     max_message_size: usize,
     grpc_config: GrpcClientConfig,
+    metrics: Arc<ExecutionPlanMetricsSet>,
+    partition: usize,
```

Comment:
looks like there is single partition as the result of this exec, so not sure if we need partition here or we could use 0 later?